lib/releases.sh

```diff
@@ -65,56 +101,55 @@ function get_latest_release() {
   fi
 }
 
-# CAPM3, CAPI and BMO release path
-CAPIRELEASEPATH="{https://api.github.com/repos/${CAPI_BASE_URL:-kubernetes-sigs/cluster-api}/releases}"
-
+CAPIGOPROXY="https://proxy.golang.org/sigs.k8s.io/cluster-api/@v/list"
 # CAPM3, CAPI and BMO releases
 if [ "${CAPM3RELEASEBRANCH}" = "release-1.6" ]; then
   # 1.6.99 points to the head of the release-1.6 branch. Local override for CAPM3 is created for this version.
   export CAPM3RELEASE="v1.6.99"
-  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.6.")}"
+  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release_from_goproxy "${CAPIGOPROXY}" "v1.6.")}"
 elif [ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]; then
   # 1.7.99 points to the head of the release-1.7 branch. Local override for CAPM3 is created for this version.
   export CAPM3RELEASE="v1.7.99"
-  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.7.")}"
+  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release_from_goproxy "${CAPIGOPROXY}" "v1.7.")}"
 elif [ "${CAPM3RELEASEBRANCH}" = "release-1.8" ]; then
   # 1.8.99 points to the head of the release-1.8 branch. Local override for CAPM3 is created for this version.
   export CAPM3RELEASE="v1.8.99"
-  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.8.")}"
+  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release_from_goproxy "${CAPIGOPROXY}" "v1.8.")}"
 else
   # 1.9.99 points to the head of the main branch of CAPM3. Local override for CAPM3 is created for this version.
   export CAPM3RELEASE="v1.9.99"
-  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.8.")}"
+  export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release_from_goproxy "${CAPIGOPROXY}" "v1.8.")}"
 fi
 
 CAPIBRANCH="${CAPIBRANCH:-${CAPIRELEASE}}"
 
 # On first iteration, jq might not be installed
 if [[ "$CAPIRELEASE" == "" ]]; then
-  command -v jq &>/dev/null && echo "Failed to fetch CAPI release from Github" && exit 1
+  command -v jq &>/dev/null && echo "Failed to fetch CAPI release from Github"
```

Comment:
```suggestion
  echo "Failed to fetch CAPI release from GOPROXY, now trying to fetch with Github"
```
we don't need `command -v jq &>/dev/null` , not sure why this was here before.